ajax/libs/jclic/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "jclic",
+  "title": "JClic.js",
+  "description": "HTML5 player for JClic activities",
+  "version": "0.1.43b",
+  "homepage": "http://projectestac.github.io/jclic.js",
```

Comment:
@Awmusic12635 It's better to use `https` instead of `http` for `github.io` domain.